bundles/org.openhab.binding.tradfri/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -176,11 +176,129 @@
 		<config-description-ref uri="thing-type:tradfri:device"/>
 	</thing-type>
 
+	<thing-type id="0007">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Air Purifier device</label>
```

Comment:
```suggestion
		<label>Air Purifier Device</label>
```

or perhaps just "Air Purifier"?

Reply:
Opted for the shortest form "Air Purifier". Thanks.